.vscode/settings.json

```diff
@@ -8,7 +8,6 @@
         ".github": false,
         "**/.pytest_cache": true,
         "**/__pycache__**": true,
-        "web/node_modules": true,
```

Comment:
i hate to push my way of life on others. but this is really annoying and i can't override. i want to go in node_modules pretty regularly from vscode. do yall have strong feelings about it? individuals can add it to their vscode settings if they want. but reverse is not true.